docs/how-to/report-security-vuln.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Report security vulnerabilities
+===============================
+
+
+How to report
+-------------
+
+*Please don't open a public GitHub issue for security problems.*
+
+1. *Open a Private Security Report* in the MicroCeph repository `here <https://github.com/canonical/microceph/security/advisories>`_
```

Comment:
- The use of numbers and bullet points can be confusing - I'd recommend sticking to bullet points.
- Also, let's use a more descriptive link here for accessibility. I suggest adding the link to "in the MicroCeph repository", and removing "here".